advanced/fix_files.sh

```diff
@@ -0,0 +1,15 @@
+# Touch files to ensure they exist (create if non-existing, preserve if existing)
+touch /var/log/pihole-FTL.log /var/log/pihole.log
+touch /run/pihole-FTL.pid /run/pihole-FTL.port
+touch /etc/pihole/dhcp.leases
+mkdir -p /var/run/pihole
+mkdir -p /var/log/pihole
```

Comment:
Can probably clean this up with a few changes.

```suggestion
mkdir -p /var/{run,log}/pihole
```